frontend/src/metabase/notifications/modals/CreateOrEditQuestionAlertModal/CreateOrEditQuestionAlertModal.tsx

```diff
@@ -251,12 +250,12 @@ export const CreateOrEditQuestionAlertModal = ({
           <Modal.Title>{isEditMode ? t`Edit alert` : t`New alert`}</Modal.Title>
           <Modal.CloseButton />
         </Modal.Header>
-        <Modal.Body p="2.5rem">
-          <Stack gap="2.5rem">
+        <Modal.Body p="2.5rem" pt="0" pb="2rem">
+          <Stack gap="2rem">
```

Comment:
Just curious, why are you not using spacing units here  (e.g. `xl`, which is `2rem`)? 

It seems like a general concern regarding spacing options in mockups vs implementation. In the ideal world we should have a list of possible spacing options (re-defined or added to the Mantine theme) that designers are aware of, but I'm not sure if we have the capacity to nitpick and follow a practice like this :)